R/benchmarks.R

```diff
@@ -2,25 +2,24 @@
 #'
 #' @param run_id the run_id to get benchmarks for (default: `NULL`, list all)
 #' @param batch_id the batch_id to get benchmarks for (default: `NULL`, list all)
+#' @param run_reason a string to specify the run reason (default: `NULL`, list all)
 #' @inheritParams runs
 #'
 #' @return the response
 #' @export
-benchmarks <- function(run_id = NULL, batch_id = NULL, simplifyVector = TRUE, flatten = TRUE, ...) {
+benchmarks <- function(run_id = NULL, batch_id = NULL, run_reason = NULL, simplifyVector = TRUE, flatten = TRUE, ...) {
   req <- req_url_path_append(conbench_request(), "benchmarks")
```

Comment:
Similarly, on `main` [the preferred path here will now be `benchmark-results`](https://github.com/conbench/conbench/blob/094251b041c7615637172ff789ce736bb4a80630/conbench/api/results.py#L254-L278), though `benchmarks will continue to work. It doesn't look like that's deployed to arrow conbench yet, though I think JP is going to do so before long.